packages/iov-lisk/src/liskconnection.spec.ts

```diff
@@ -32,6 +32,7 @@ import { Ed25519Wallet, UserProfile } from "@iov/keycontrol";
 import { pubkeyToAddress } from "./derivation";
 import { liskCodec } from "./liskcodec";
 import { generateNonce, LiskConnection } from "./liskconnection";
+import { toListPromise } from "@iov/stream";
```

Comment:
Lets always use those types of import blocks separated by an empty line:

1. external packages
2. @iov/ packages
3. local file imports